dev_tools/packaging/verify-published-package.sh

```diff
@@ -61,13 +61,9 @@ cd "${tmp_dir}"
 trap "{ rm -rf ${tmp_dir}; }" EXIT
 
 # Test both the python 2 and python 3 versions.
```

Comment:
update comment

Reply:
Opps merged too soon. Will correct!

Reply:
Prefer to defer to the author of a PR as the one to merge it, unless it's blocking another PR or of course if they aren't able to merge it in the first place. The author is in the best position to know if it's ready or not.

I got into the habit of doing the merges for people because most other contributors can't merge, and because I usually have a good idea of if it's ready or not. I'll try to tone that down.